index.html

```diff
@@ -2,188 +2,242 @@
+                        dummy text.
+                    </p>
                     <ul>
+                        <li>CSS</li>
                         <li>HTML</li>
                         <li>Bootstrap</li>
                         <li>Ruby</li>
                     </ul>
+                    <button class="btn">See Project</button>
                 </div>
-                <button>See project</button>
             </div>
 
-            <div class="card">
-                <div class="card-content">
-                    <h2>Profesional Art Printing Data</h2>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
-                        industry's standard.</p>
-                    <ul>
-                        <li>HTML</li>
-                        <li>Bootstrap</li>
-                        <li>Ruby</li>
-                    </ul>
+            <div class="my-cards">
+                <div class="card">
+                    <div class="card-content hide">
+                        <img class="first-image" src="/images/Img Placeholder.png" alt="" />
+                        <h2>Professional Art Printing Data</h2>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts
+                            or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li>HTML</li>
+                            <li>Bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="btn-show">See project</button>
                 </div>
-                <button class="btn">See project</button>
-            </div>
 
-            <div class="card">
-                <div class="card-content">
-                    <h2>Profesional Art Printing Data</h2>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
-                        industry's standard.</p>
-                    <ul>
-                        <li>HTML</li>
-                        <li>Bootstrap</li>
-                        <li>Ruby</li>
-                    </ul>
+                <div class="card bg-2">
+                    <div class="card-content">
+                        <h2>Professional Art Printing Data</h2>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts
+                            or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li>HTML</li>
+                            <li>Bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
```

Comment:
[OPTIONAL]
- [ ] I would kindly suggest you work on the the first card of the work cards as the `see project` button is not aligning perfectly with the card as seen in the Figma design, and currently in your design the `width` and `height` of the first card is different from the other cards. Kindly check this and work on it 
  - N.B: I actually marked this as OPTIONAL because the previous reviewer did not raise any issue concerning it. Please i would highly advise you implement it.

| Your Design 👇 | Figma Design 👇 |
| ------------- | ------------- |
| ![kem](https://user-images.githubusercontent.com/78634841/230732533-3ce36ceb-e5f9-4a7e-a193-990acff7f6bf.png) | ![image](https://user-images.githubusercontent.com/78634841/230732585-80892335-2b98-4083-a0df-401615ad3f57.png) |

Reply:
I've fixed the alignment of the first card